src/server/index.test.jsx

```diff
@@ -1106,11 +1106,6 @@ describe('Server', () => {
     });
 
     describe('for home pages', () => {
-      it('should respond with JSON', async () => {
```

Comment:
we don't use json routes with home pages now

Reply:
We do on local though

Reply:
I think the JSON routes are still used for the local fixture routes on dev